pkg/build/strategies/onbuild/onbuild.go

```diff
@@ -49,17 +53,20 @@ func New(config *api.Config, overrides build.Overrides) (*OnBuild, error) {
 		tar:    tar.New(),
 	}
 	// Use STI Prepare() and download the 'run' script optionally.
-	s, err := sti.New(config, overrides)
+	// NOTE: update build status?
```

Comment:
Not obvious to me exactly what this entails ... please elaborate.  Which struct/field specifically could be updated?